.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
 ## Description
+
+<!-- Every PR should have a corresponding issue that describes and motivates the work done in the PR -->
+closes: <!-- Link issue here -->
```

Comment:
```suggestion
closes <!-- Link issue here -->
```

Uncertain if the colon would break the [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) so IMO we should just remove it